composer.json

```diff
@@ -7,7 +7,7 @@
     "php": ">=7.3",
     "psr/http-message": "^1.0.0",
     "spryker-shop/shop-cms-slot-extension": "^1.0.0",
-    "spryker/category-storage": "^1.11.0",
+    "spryker/category-storage": "^1.11.0 || ^2.0.0",
```

Comment:
Please drop the nullable support for `$storeName` argument in the `CoremediaToCategoryStorageClientBridge:: getCategoryNodeById()` method.